migrations/2021-04-24-115720_remove_version_authors/down.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE version_authors(
+    id SERIAL  NOT NULL CONSTRAINT version_authors_pkey PRIMARY KEY,
+    version_id INTEGER NOT NULL CONSTRAINT fk_version_authors_version_id REFERENCES versions ON DELETE CASCADED,
```

Comment:
There is an error in this query:

```suggestion
    version_id INTEGER NOT NULL CONSTRAINT fk_version_authors_version_id REFERENCES versions ON DELETE CASCADE,
```